packages/fxa-settings/src/components/Settings/ConnectedServices/index.tsx

```diff
@@ -239,6 +240,18 @@ export const ConnectedServices = () => {
           </Localized>
         </div>
 
+        <div className="mt-5 text-center mobileLandscape:text-start">
+          <Localized id="cs-recent-activity">
```

Comment:
You can use the new `FtlMsg` instead of `Localized`. We have a lot of testing set up to cover l10n tests automatically, but there's one or two bugs in the l10n improvements epic I hope we'll get to next sprint to enable them.